### PR TITLE
docs: update guide of Wave generator in strings.xml

### PR DESCRIPTION
**Pull request by KushMehta1511**
In the guide of Wave Generator in the third line, it is written "allow us to changes" which should be "allows us to change".

**Changes**: 
In line 799 of strings.xml "allow us to changes" is changed to "allows us to change".

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -796,7 +796,7 @@
     <string name="hide_guide_text">Hide Guide</string>
 
     <string name="wave_gen_guide_intro">The wave generator can be used to generate different types of waves like
-        Sine wave, Square wave, Saw Tooth wave and allow us to changes their characteristics like frequency, phase, duty.
+        Sine wave, Square wave, Saw Tooth wave and allows us to change their characteristics like frequency, phase, duty.
```
Comment (CodeWithSouma) [inline app/src/main/res/values/strings.xml:799]: This is a simple mistake try to fix some big issue.

**Review by CodeWithSouma — APPROVED** [review]

**Comment by KushMehta1511** [comment]
@CodeWithSouma Ok thank you, sir, for the review

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (KushMehta1511) [reply app/src/main/res/values/strings.xml:799]: Ok, thank you, sir, for the review.

Comment (CloudyPadmal) [inline app/src/main/res/values/strings.xml:799]:
```suggestion
        Sine wave, square wave and saw-tooth wave allow us to change their characteristics like frequency, phase and duty.
```

**Review by CloudyPadmal — CHANGES_REQUESTED** [review]
Let's make it a bit better than doing just a word

**Review by CloudyPadmal — APPROVED** [review]

**Pull request merged**